sycl/test-e2e/Reduction/reduction_span.cpp

```diff
@@ -3,7 +3,7 @@
 
 // Windows doesn't yet have full shutdown().
 // UNSUPPORTED: ze_debug && windows
-
+// REQUIRES: aspect-usm_shared_allocations
```

Comment:
Same here. Can we rewrite it using buffers instead?

Reply:
I think we do need some USM allocation for the `span` version of the reduction. However, using device allocations might make this test supported on a broader range of devices.